tensorbay/geometry/vector.py

```diff
@@ -50,44 +50,42 @@ class Vector(UserSequence[float]):
 
     _DIMENSION: Optional[int] = None
 
-    def __new__(  # type: ignore[misc]
+    def __new__(  # type: ignore[misc]  # pylint: disable=unused-argument
         cls: Type[_V],
-        *args: Union[None, float, Iterable[float]],
-        **kwargs: float,
+        x: float,
+        y: float,
+        z: Optional[float] = None,
+        v: Optional[int] = None,
```

Comment:
it is awkward to add `v` here, do we have different solutions?

Reply:
May be we can override the `__new__` method for `Keypoint2D`, and add `v` there.